packages/core/saved-objects/core-saved-objects-server/index.ts

```diff
@@ -64,3 +65,22 @@ export type {
 } from './src/serialization';
 export type { ISavedObjectTypeRegistry } from './src/type_registry';
 export type { SavedObjectsValidationMap, SavedObjectsValidationSpec } from './src/validation';
+export type { ISavedObjectsEncryptionExtension, EncryptedObjectDescriptor } from './src/encryption';
+export type {
+  CheckAuthorizationParams,
+  AuthorizationTypeEntry,
+  AuthorizationTypeMap,
+  CheckAuthorizationResult,
+  EnforceAuthorizationParams,
+  AddAuditEventParams,
+  RedactNamespacesParams,
+  ISavedObjectsSecurityExtension,
+} from './src/security';
+export { AuditAction } from './src/security';
+export type { ISavedObjectsSpacesExtension } from './src/spaces';
+export type { SavedObjectsExtensions } from './src/extensions';
+export {
+  ENCRYPTION_EXTENSION_ID,
+  SECURITY_EXTENSION_ID,
+  SPACES_EXTENSION_ID,
+} from './src/extensions';
```

Comment:
I'm not sure. What would that provide?

Reply:
@jeramysoucy we'd want to export those constants from `src/core/server/index.ts` if they're used elsewhere in Kibana. Alternatively, I guess we could import them from the package if needed.

Reply:
Ah right...I see there are in-line constant strings that should be replaced in x-pack. I'll export these as Pierre suggested and replace all of the in-line strings with the defined constants.

Reply:
I am reverting this change to see if it resolves the issues I am seeing in CI. @mistic believed this could have been causing a problem - "I think the problem is that you are consuming things other than types from core in the test system...why dont consume those directly from the package itself?".

I also want to make sure I understand the original intent. I originally searched the codebase for static strings and replaced then with references to the declared constants (exported from `src/core/server/index.ts`), but perhaps this is not what @pgayvallet intended.

Reply:
@pgayvallet @TinaHeiligers consuming non-types from core was the issue. I will leave the imports of these constants as references to the package, as Tiago suggested.